.github/workflows/learnings-reuse-weekly.yml

```
@@ -0,0 +1,67 @@
+          if git diff --quiet; then
+            echo "No changes to commit"
+            exit 0
+          fi
+          
+          git add docs/micro-lessons/
+          git commit -m "chore: update micro-lesson reuse counts (weekly automation)
+
+          🤖 Generated with GitHub Actions weekly cron
+
+          Co-Authored-By: GitHub Actions <action@github.com>"
+          git push
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)